assets/app/views/directives/osc-key-values.html

```diff
@@ -2,18 +2,27 @@
     <div class="form-inline labels-edit" ng-show="editable">
       <form class="edit-label" name="form" novalidate>
         <div class='form-group' ng-class="{'has-error': form.key.$error.oscKeyValid}">
-          <input class="form-control" type="text" name="key" placeholder="{{keyTitle}}" ng-model="key" 
+          <input class="form-control"
+                 type="text"
+                 name="key"
+                 placeholder="{{keyTitle}}"
+                 ng-model="key"
+                 ng-keyup="toggleAdd()"
                  osc-input-validator="key">
         </div>
         <div class="form-group" ng-class="{'has-error': form.value.$error.oscValueValid}">
```

Comment:
make the add bundle enablement trigger off of the form's $invalid property, rather than explicitly binding something to `keyup` (see create.html for an example)